coldfront/core/project/views.py

```diff
@@ -443,6 +443,17 @@ def get_context_data(self, **kwargs):
         context['user_agreement_signed'] = \
             access_agreement_signed(self.request.user)
 
+        compute_allocation_information = []
+        for project in project_list:
+            try:
+                information = get_project_compute_allocation(project).get_information
+                information = information[len('Service Units: '):-len(' <br>')]
+                compute_allocation_information.append(information if information else 'N/A')
```

Comment:
Any reason to do this differently from the other view? `project.compute_allocation_information = ...`